lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -1639,6 +1682,11 @@ def main():
                                      launch_template_id=dict(type='str'),
                                  ),
                                  ),
+            mixed_instances_policy=dict(type='dict',
+                                        default=None,
+                                        options=dict(
+                                            instance_types=dict(type='list'),
```

Comment:
```suggestion
                                            instance_types=dict(type='list', elements='str'),
```
It's good to also tell people that you're expecting a list of "what".  (Hmm the sanity tests really should have caught this)